app/src/main/java/org/dhis2/usescases/programEventDetail/ProgramEventDetailLiveAdapter.kt

```diff
@@ -35,9 +42,41 @@ class ProgramEventDetailLiveAdapter(
     }
 
     override fun onBindViewHolder(holder: EventViewHolder, position: Int) {
-        holder.bind(getItem(position), null) {
-            getItem(holder.bindingAdapterPosition)?.toggleValueList()
-            notifyItemChanged(holder.bindingAdapterPosition)
+        getItem(position)?.let {
+            val materialView = holder.itemView.findViewById<ConstraintLayout>(R.id.materialView)
+            materialView.visibility = View.GONE
+            val composeView = holder.itemView.findViewById<ComposeView>(R.id.composeView)
+            composeView.setContent {
```

Comment:
will we still allow the option of using the old design here?

Reply:
No, we are not, neither in dataset. In search is different because we reuse it for relationships, which we'll need to change in the future. I didn't remove the adapters, holders and recyclerviews because the issue will be too big and too many changes. I replace just the item. My idea is to create separate issues to remove all xml vies in these screens.